bin/cylc-scan

```diff
@@ -253,6 +266,36 @@ def main():
                 print "%s|%s|%s|%s|%s" % (name, owner, host, key, state_line)
             continue
 
+        if options.json_format:
+            suite_inf = {"owner": owner, "host": host, "port": port}
+            for key in [KEY_TITLE, KEY_DESCRIPTION, KEY_UPDATE_TIME]:
+                try:
+                    value = suite_identity[KEY_META].get(key)
+                except KeyError:
+                    # KEY_UPDATE_TIME and Compat:<=7.5.0
+                    value = suite_identity.get(key)
+                if value:
+                    suite_inf.update({key: ' '.join([x.strip() for x in
+                                      str(value).split('\n') if x])})
+                totals = suite_identity.get(KEY_STATES)
+                if totals is None:
+                    continue
+                point_state_lines = get_point_state_count_lines(
+                    *totals, use_color=options.color)
```

Comment:
er?

Reply:
Nice spot! I did ensure ``--json``  cannot be used with the ``--color`` option on lines 185-188 (in case anyone wanted to check) but I agree it is best to just remove this argument as it is therefore unnecessary and we can then trim the line count by 1. Will update in a minute.

Reply:
I also spotted this in the equivalent line within the ``--raw`` option encompassing statement and removed that too in my recent commit, as colour shouldn't (though strictly can't anyway, via parser error) be used with that either.